commands/system/update.sh

```diff
@@ -18,6 +18,31 @@ function system:update() {
     docker pull mikefarah/yq
     ${DOCKER_COMPOSE} build --pull
 
+
+    cd services/conf.d
+
+    for f in *; do
+        if [ -f ${ROOT_DIR}/services/${f}/docker-compose.yml ]; then
+            _logYellow "Update service ${f}"
+            cd ${ROOT_DIR}/services/${f}
+            ${DOCKER_COMPOSE} pull
+            ${DOCKER_COMPOSE} build --pull
+        fi
+    done
+
+    cd ${ROOT_DIR}
+
+    _ddeCheckNetwork
+
+    system:services:update dnsmasq
+    system:services:enable dnsmasq
```

Comment:
@rugbymauri the update command should only execute :update, not :enable. :enable should be handled by the up command.


Also we should get the list of services dynamically, in case a new service gets added